bio/mtnucratio/wrapper.py

```
@@ -0,0 +1,37 @@
+# output file names. Moving to a temporary directory:
+with TemporaryDirectory() as tempdir:
+    # Symlink input file to control output file names
+    link_path = f"{tempdir}/{basename(bam)}"
+    bam_path = realpath(bam)
+    symlink(bam_path, link_path)
+
+    # Run MTNucRatioCalculator
+    shell("mtnucratio {link_path} {chrom} {log}")
+
+    # Rename output files
+    txt = snakemake.output.get("txt")
```

Ruff (0.7.0)

Undefined name `snakemake`

(F821)